### PR TITLE
Luke quickstart update

### PR DESCRIPTION
**Pull request by cyniphile**
Update to quickstart to smooth learning process and reflect key changes in UX.

### CODE REVIEW

File: docs/Quickstart.mdx [hunk 1]
```diff
@@ -42,67 +47,79 @@ The <div className="button gray_button"><UserOutlined /> Sign In</div> button sh
...
+
+Click on the downward-pointing carat <DownOutlined className="button darkblue_button_icon"/> in the corner of the <div className="button darkblue_button">**Excel_Read**</div> node to expand it and edit its attributes.
 
-<img alt="Quickstart Flow Init" src="https://ganymede-bio.mo.cloudinary.net/apiServer/QuickstartFlow_20230405.png"/>
+<div class="text--center">
+<img alt="Quickstart flow description" src="https://ganymede-bio.mo.cloudinary.net/apiServer/QuickstartAddedNodeExcel2_20230501.png"/>
+</div>
 
-Node attributes with <div className="green_text">**green labels**</div> indicate validation attributes.  In this example, the <div className="green_text">**excel**</div> node attribute on <NodeChip text="Excel_Read" /> refers to the file extension that the *flow* looks for during runs.  Click on the edit button, and modify the <div className="green_text">**excel**</div> node attribute to be ".xlsx", so that only files with ".xlsx" as an extension would be let through.  
+Change the **excel** node attribute from "excel" to ".xlsx" so that only files with `.xlsx` as an extension will be processed by this node (other file types, such as `.xlsm` will be blocked).
```
Comment (sbelkadi) [inline docs/Quickstart.mdx:83]: Below this, we should add a note about changing the "results" parameter to Quickstart_Absorbance_Change_Excel_Read_results, which is mentioned later in the SQL query.

File: docs/Quickstart.mdx [hunk 2]
```diff
@@ -137,17 +160,32 @@ Upon doing so, the response in the Save Pipeline Cell should say "Commit complet
 
 ### Step 6: Run the flow
 
-Deployment takes roughly 1-2 minutes.  Once the deployment is completed, return to the <div className="button darkblue_button"><BranchesOutlined /> **Flow Editor**</div> view and click <div className="button darkblue_button_mui"><ExpandLessOutlined className="white_icon" />**Expand**</div> in the Run Queue bar at the bottom of the screen to expand the input form for running nodes.  Select local storage and the PlateReader.xlsx file downloaded at the beginning of this quickstart.
+Deployment takes roughly 1-2 minutes.  Once the deployment is completed, return to the <div className="button darkblue_button"><BranchesOutlined /> **Flow Editor**</div> view and click <div className="button darkblue_button_mui"><ExpandLessOutlined className="white_icon" />**Expand**</div> in the Run Queue bar at the bottom of the screen to expand the input form for running nodes. Click the "Drag and drop files here or Browse" button and upload the "PlateReader.xlsx" file downloaded at the beginning of this quickstart.
```
Comment (sbelkadi) [inline docs/Quickstart.mdx:163]: We should add a note here that the notification page can be used to track the progress of a Save, and a toast message will pop up in the UI when the Save is complete.

Comment (cyniphile) [reply docs/Quickstart.mdx:163]: sg will do

File: docs/Quickstart.mdx (hunk 1 above)
Comment (cyniphile) [reply docs/Quickstart.mdx:83]: yeah just tested again, this field auto-populates correctly, assuming the user named the flow "Quickstart..." as described.

**Review by sbelkadi — APPROVED** [review]

**Pull request merged**